Ruby-Assignment-8-TDD/roman-number-coding/lib/roman_code_evaluator.rb

```diff
@@ -0,0 +1,38 @@
+
+class RomanCodeEvaluator
+  @@CODE = { coin: 'I', dollar: 'V', yen: 'X', rupee: 'L' }
+
+  attr_reader :multiplicatives
+
+  def initialize
+    @multiplicatives = Hash.new
+    eval_multiplicative("coin coin Silver 34")
```

Comment:
Actually its none of the concern of TDD assignment , but do not call method from initialize, You can call it by the instance of the same class.